### DISCUSSION TITLE
참가등록 수정 부탁드립니다

### DISCUSSION THREAD

**Discussion by thursdaydreaming** [devrel-kr/HackaLearn#33] [1 Q&A - HackaLearn 운영] [answered]
1. 팀으로 등록한 후 팀원이 각자 등록해야 함을 알았습니다. sns 아이디 적을 때 모든 팀원의 아이디를 한번에 적었고, 심지어 이메일을 잘못 기재해서 한번 더 등록했는데요, 중복으로 신청하게 되어 혹시 처음 신청한 내역을 삭제조치 해주실 수 있나요? ㅠㅠ 감사합니다

**Comment by justinyoo** [comment, marked as answer]
@thursdaydreaming 등록한 팀명 알려주세요

**Reply by thursdaydreaming** [reply]
godlife 팀입니다. 혹시 hyeminseo42@gmail.com 으로 등록했을 때 이메일이 오지 않는데, 한 번 확인 부탁드려도 될까요?

**Reply by justinyoo** [reply]
https://github.com/devrel-kr/HackaLearn/blob/main/teams/godlife.md 이 페이지 확인해 보시겠어요?

**Reply by justinyoo** [reply]
이메일은은 잘 갔습니다. (19시 48분) 스팸메일 박스 확인해 보시겠어요?

![image](https://user-images.githubusercontent.com/1538528/127851530-f1aa6211-3f55-4cd2-a1ab-884c0e821dce.png)

**Reply by thursdaydreaming** [reply]
@justinyoo 네 확인 감사드립니다. 팀원 문제 없습니다. 이메일도 확인해보겠습니다

**Reply by thursdaydreaming** [reply]
@justinyoo 아 제가 스팸함을 헷갈렸네요.. 다시한번 확인 감사드립니다!

**Reply by justinyoo** [reply]
@thursdaydreaming 답변 채택 좀... 굽신
